### ISSUE TITLE
Arch install package lack required signature

### ISSUE THREAD

**Issue by senpl** [hovancik/later-on#10]
I tried to install on arch linux with command:

`sudo pacman -U https://github.com/hovancik/later-on/releases/download/v0.0.3/LaterOn-0.0.3.pacman`

I got following error:

`error: '/var/cache/pacman/pkg/LaterOn-0.0.3.pacman': package lack required signature`

Any short installation instruction would be great. Arch repo would be even better.

**Comment by hovancik** [comment]
Actually I have no idea how to install it correctly ;/ Installers are created by `electron-builder` and I have no experience with Arch Linux. 

I have another `electron` app that has `aur` package https://aur.archlinux.org/packages/stretchly-bin/ so maybe it could be replicated? But again, I have no knowledge about `aur`.

**Comment by senpl** [comment]
As valid Workaround use of .AppImage works so it seems like something related to keys.

**Comment by hovancik** [comment]
I am not setting any keys for any platform/installer. Do you have Stretchly installed by any chance? I am using the same system to build installers there, but never heard any complaint there. 

Maybe there is some option to disable keys checks? On MacOS users have to open app in special way and mark it as trusted app. Keys are expensive.

**Comment by senpl** [comment]
Yes. I got streachly, it works from arch packages, not from pacman file, and there message is clear about invalid keys. Anyway on arch `yaourt -S stretchly-bin` just install it without problem.

**Comment by hovancik** [comment]
I guess we need to then wait for someone to create the same for LaterOn.
